9.3_rotbinsearch.js

```diff
@@ -0,0 +1,101 @@
+
+//finds the position of the min value in rotated increasingly sorted array
+var binSearchMin = function (arr, min, max) {
+	if (max - min < 2){
+		return 0;
+	}
+	if (arr[min] < arr[max] || (min - 1 > 0 && arr[min -1] > arr[min])) {
```

Comment:
Could do (min > 1), also arr[min -1] > arr[min] will never use arr[0] .. Is that what you meant? The condition insures that min is at least 2